### PR TITLE
Initial issue template

### PR DESCRIPTION
**Pull request by jedahan**
The issue template is very verbose, to explain why we are asking for some work up front. Not sure if this is asking too much, or if there can be better formatting. Hope this is helpful!

### CODE REVIEW

**Comment by NigoroJr** [comment]
:+1: :+1:

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+To make sure we can quickly assist you with any bugs, we have two requests:
+
+## 1. Reproduce the bug with a minimal **.zshrc**
+
+Remove as many lines in your **.zshrc** as possible, for example, if theres a problem with zsh-syntax-highlighting, try something like:
+
+```zsh
+  # Example , minimal zshrc
+  source ~/.zplug/init.zsh
+  zplug "jimmijj/zsh-syntax-highlighting"
```
Comment (NigoroJr) [inline .github/ISSUE_TEMPLATE.md]: Maybe `zsh-users/zsh-syntax-highlighting`?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Thank you for helping improve zplug!
+
+To make sure we can quickly assist you with any bugs, we have two requests:
+
+## 1. Reproduce the bug with a minimal **.zshrc**
+
+Remove as many lines in your **.zshrc** as possible, for example, if theres a problem with zsh-syntax-highlighting, try something like:
+
+```zsh
+  # Example , minimal zshrc
```
Comment (NigoroJr) [inline .github/ISSUE_TEMPLATE.md]: Very minor, but couldn't help pointing out the space before the comma :grin:

**Comment by babarot** [comment]
Thanks @jedahan @NigoroJr , merging

**Pull request merged**